src/server.cpp

```diff
@@ -47,6 +47,38 @@ namespace quicr {
 
     void Server::ResolveSubscribe(ConnectionHandle, uint64_t, const SubscribeResponse&) {}
 
+    void Server::UnbindPublisherTrack(ConnectionHandle connection_handle,
+                                      const std::shared_ptr<PublishTrackHandler>& track_handler)
+    {
+        std::unique_lock lock(state_mutex_);
+
+        auto conn_it = connections_.find(connection_handle);
+        if (conn_it != connections_.end()) {
```

Comment:
I would maybe flip the condition and return early to avoid indenting the entire function, but that's just a nit.

Reply:
makes sense, updated.